### PR TITLE
PDA Message Server can be constructed

### PR DESCRIPTION
**Pull request by JJRcop**
## About The Pull Request

I made a machine board for the PDA message server and added it to the telecomms techweb node. To keep the usefulness of the machine's password, a player built message server won't work until 15 minutes after construction. I've also converted a lot of magic numbers to defines, and changed the departmentType var on request consoles into a bitflag.

I'm open to ideas other than a time gate when it comes to keeping the server password's usefulness.

## Why It's Good For The Game

You can't build a Message Server right now. If it's destroyed, PDAs may be knocked out for the rest of the round.

## Changelog
:cl: JJRcop
add: You can now build a new PDA messaging server if you lose it.
tweak: Requests console messages no longer work if the messaging server is offline. Emergency alerts will still function in that case as long as the rest of telecomms is online.

/:cl:

### CODE REVIEW

File: code/game/machinery/requests_console.dm [hunk 1]
```diff
@@ -87,38 +100,19 @@ GLOBAL_LIST_EMPTY(allConsoles)
...
-			if(!("[department]" in GLOB.req_console_assistance))
-				GLOB.req_console_assistance += department
-			if(!("[department]" in GLOB.req_console_supplies))
-				GLOB.req_console_supplies += department
-			if(!("[department]" in GLOB.req_console_information))
-				GLOB.req_console_information += department
+
+	if(departmentType)
+
+		if((departmentType & REQ_DEP_TYPE_ASSISTANCE) && !("[department]" in GLOB.req_console_assistance))
```
Comment (AnturK) [inline code/game/machinery/requests_console.dm]: why are you checking "[department]" here but adding raw, is it text or object

**Review by AnturK — CHANGES_REQUESTED** [review]
Just some nitpick

File: code/game/machinery/requests_console.dm (hunk 1 above)
Comment (JJRcop) [reply code/game/machinery/requests_console.dm]: Silly me, I can just check raw cause it's always text

**Comment by JJRcop** [comment]
I will update kilostation if it's merged before this... Actually do not merge if #43876 is merged first

**Pull request merged**